### DISCUSSION TITLE
how to use with github pages

### DISCUSSION THREAD

**Discussion by siqnole** [animate-css/animate.css#1430] [Q&A] [answered]
Im making a website with github pages, and im wondering how I can impliment this into my site.

**Comment by KodingKhurram** [comment, marked as answer]
You can just do it as you do in normal webpages.

1. Include `animate.css` library
     `<link
    rel="stylesheet"
    href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css"
  />`
2. Add animation classes.
3. There you go.
